openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh

```diff
@@ -0,0 +1,296 @@
+if [ "$#" -lt 8 ]; then
+  echo "Incorrect number of arguments $#, expecting 8:"
+  echo "$0 with arguments: "
+  echo "   1. MASTER_PUBLIC_DNS"
+  echo "   2. STANDALONE_ETCDS_PRIVATE_DNS"
+  echo "   3. PBENCH_COLLECTION_INTERVAL"
+  echo "   4. PBENCH_RESULTS_DIR_NAME"
+  echo "   5. CLUSTER_LOADER_CONFIG_FILE"
+  echo "   6. WAIT_TIME_BEFORE_STOPPING_PBENCH"
+  echo "   7. PBENCH_REGISTER"
+  echo "   8. PBENCH_RESULTS_INTERNAL_HOSTANME"
+  exit 0
+fi
+
+echo -e "\nCurrent environment variables:\n$(env)"
+echo -e "\nCurrent bash shell options: $(echo $-)"
+
+DEFAULT_COLLECTION_DURATION_SECS=10
+DEFAULT_PBENCH_REGISTER=false
+
+MASTER_PUBLIC_DNS=$1
+STANDALONE_ETCDS_PRIVATE_DNS=$2
+PBENCH_COLLECTION_INTERVAL=$3
+PBENCH_RESULTS_DIR_NAME=$4
+CLUSTER_LOADER_CONFIG_FILE=$5
+WAIT_TIME_BEFORE_STOPPING_PBENCH=${6:-$DEFAULT_COLLECTION_DURATION_SECS}
+PBENCH_REGISTER=${7:-$DEFAULT_PBENCH_REGISTER}
+PBENCH_RESULTS_INTERNAL_URL_PREFIX=$8
+
+echo "MASTER_PUBLIC_DNS from first argument: ${MASTER_PUBLIC_DNS}"
+echo "STANDALONE_ETCDS_PRIVATE_DNS from second argument: ${STANDALONE_ETCDS_PRIVATE_DNS}"
+echo "PBENCH_COLLECTION_INTERVAL from third argument: ${PBENCH_COLLECTION_INTERVAL}"
+echo "PBENCH_RESULTS_DIR_NAME from fourth argument: ${PBENCH_RESULTS_DIR_NAME}"
+echo "CLUSTER_LOADER_CONFIG_FILE from fifth argument: ${CLUSTER_LOADER_CONFIG_FILE}"
+echo "WAIT_TIME_BEFORE_STOPPING_PBENCH from sixth argument: ${WAIT_TIME_BEFORE_STOPPING_PBENCH}"
+echo "PBENCH_REGISTER from seventh argument:  ${PBENCH_REGISTER}"
+
+
+echo -e "\nChecking current version of atomic-openshift-clients on this Test Client instance:"
+yum list installed atomic-openshift-clients
+yum clean all
+yum list atomic-openshift-clients
+
+### TO-DO: 
+### - update the atomic-openshift-clients rpm to latest or to specified version
+
+openshift version
+oc version
+echo -e "\nRemoving current /root/.kube dir on this host"
+rm -rf /root/.kube
+
+ls -ltra /root
+echo -e "Creating a new /root/.kube dir on this host "
+mkdir -p /root/.kube
+ls -ltr /root/.kube/config
+
+echo -e "SCP the /etc/origin/master/admin.kubeconfig file from Master node ${MASTER_PUBLIC_DNS} to this Test Client host"
+scp root@${MASTER_PUBLIC_DNS}:/etc/origin/master/admin.kubeconfig /root/.kube/config
+echo -e "Checking the newly copied /root/.kube/config file on this host: \n"
+
+ls -ltr /root/.kube/config
+cat /root/.kube/config
+
+# Next step need to SCP from master the /opt/pbench-agent/config/pbench-agent.cfg, to support or test instance
+# where we will run the pbench-agent.  This has config details for where to copy pbench results after test is done
+
+echo -e "\nSCP the /opt/pbench-agent/config/pbench-agent.cfg file from Master node ${MASTER_PUBLIC_DNS} to this Test Client host"
+cp /opt/pbench-agent/config/pbench-agent.cfg /opt/pbench-agent/config/pbench-agent.cfg_ORIG
+scp root@${MASTER_PUBLIC_DNS}:/opt/pbench-agent/config/pbench-agent.cfg /opt/pbench-agent/config/pbench-agent.cfg
```

Comment:
why doesnt the ansible-host have the pbench config baked in?

Reply:
The ansible-host does have the config baked in, this is only needed if you want to run pbench from master when a test client or pbench-controller host is not available